tools/CreateDebugNuget.ps1

```diff
@@ -45,23 +56,31 @@ $tempSourceDir = Join-Path $TempDir "src"
 New-Item -ItemType "directory" -Path $tempSourceDir
 Copy-Item -Path $buildDir -Destination $tempBuildDir -Recurse
 
+#source nuspec file paths
 $sourceRDNuspec = Join-Path $sourceDir "ReactiveDomain.Debug.nuspec"
 $sourceRDTestNuspec = Join-Path $sourceDir "ReactiveDomain.Testing.Debug.nuspec"
 $sourceRDUINuspec = Join-Path $sourceDir "ReactiveDomain.UI.Debug.nuspec"
 $sourceRDUITestNuspec = Join-Path $sourceDir "ReactiveDomain.UI.Testing.Debug.nuspec"
+$sourceRDIdentityStorageNuspec = Join-Path $sourceDir "ReactiveDomain.IdentityStorage.Debug.nuspec"

```

Comment:
Reminder: This script won't work until you get that nuspec file in the reactiveDomain repo. I mention because I don't see a PR in that repo